### PR TITLE
CVE services 2.1.0

### PR DESCRIPTION
**Pull request by mprpic**
A PR of all changes done on the `cve-services-2.1.0` branch that can now be merged to master for the 1.0.0 release.

### CODE REVIEW

File: man/cve.1 [hunk 1]
```diff
@@ -30,12 +30,24 @@ Confirm create/update actions before execution (env var: CVE_INTERACTIVE)
...
+  See \fBcve-publish(1)\fP for full documentation on the \fBpublish\fP command.
+.PP
+\fBreject\fP
+  Reject a CVE record for a reserved or...
+  See \fBcve-reject(1)\fP for full documentation on the \fBreject\fP command.
+.PP
+\fBundo-reject\fP
+  Move a rejected CVE ID without a record...
+  See \fBcve-undo-reject(1)\fP for full documentation on the \fBundo-reject\fP command.
+.PP
```
Comment (juspence) [inline man/cve.1:44]: Did we miss "move-to-rejected" in docs?

File: pyproject.toml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 [tool.black]
 line-length = 100
 
+[tool.isort]
+profile = "black"
+
 [tool.mypy]
+python_version = "3.10"
```
Comment (juspence) [inline pyproject.toml:8]: This is probably too late in the game to care, but just noticed here. Does using 3.10 cause issues for people without that Python version? I know some of our tests still expect / use Python 3.6.

**Review by juspence — APPROVED** [review]

File: man/cve.1 (hunk 1 above)
Comment (mprpic) [reply man/cve.1:44]: Which "docs" do you mean? The README file?

Comment (juspence) [reply man/cve.1:44]: This man/cve.1 file here - I see a diff with lines added for all other commands, but not for undo reject.

Comment (mprpic) [reply man/cve.1:44]: That's only an interface method, the `reject` command can take no record and works as the reserved-to-rejected without a record state switch. `undo-reject` is the only special command that has to support the transition back to reserved.

**Pull request merged**